manifests/params.pp

```diff
@@ -153,31 +149,13 @@
       $config_file         = '/etc/my.cnf'
       $includedir          = '/etc/my.cnf.d'
       $datadir             = '/var/lib/mysql'
-      $log_error           = $::operatingsystem ? {
```

Comment:
Already a conditional above, so removing this pattern and adding to the correct block above.